var/spack/repos/builtin/packages/cdecimal/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Cdecimal(AutotoolsPackage):
+    """cdecimal is a fast drop-in replacement for the decimal module in
+    Python's standard library."""
+
+    homepage = "https://www.bytereef.org/mpdecimal/"
+    url      = "https://www.bytereef.org/software/mpdecimal/releases/cdecimal-2.3.tar.gz"
+
+    version('2.3', sha256='d737cbe43ed1f6ad9874fb86c3db1e9bbe20c0c750868fde5be3f379ade83d8b')
+
+    patch('darwin_install_name.patch', when='platform=darwin')
```

Comment:
Even with this patch, I'm still not able to get `cdecimal` to install on macOS.

Python's `configure` script has the following option:
```
--with-system-libmpdec  build _decimal module using an installed libmpdec
                          library
```
I'm still not sure if this is referring to `cdecimal` or `mpdecimal`, but since I wasn't able to get either to install on macOS, we'll just have to build using the vendored copy that comes with Python.

Reply:
Looks like this doesn't build on CentOS 7 either. The unit tests crash with:
```
==> 'make' '-j36' 'check'
cd tests && make "CFLAGS=-Wall -W -Wno-unknown-pragmas -DCONFIG_64 -DASM -O2 -fpic -s" "LDFLAGS=" && ./runshort.sh
make[1]: Entering directory `/scratch/ajstewart/spack-stage/spack-stage-20gAT2/cdecimal-2.3/tests'
make[1]: *** No rule to make target `runtest.c', needed by `runtest'.  Stop.
make[1]: Leaving directory `/scratch/ajstewart/spack-stage/spack-stage-20gAT2/cdecimal-2.3/tests'
make: *** [check] Error 2
```
If you skip those, installation fails with:
```
==> 'make' '-j36' 'install'
mkdir -p /blues/gpfs/home/ajstewart/spack/opt/spack/linux-centos7-x86_64/gcc-7.1.0/cdecimal-2.3-72tmlpbrxaawnmjzh2dicis3o6i54jne/include && cp mpdecimal.h /blues/gpfs/home/ajstewart/spack/opt/spack/linux-centos7-x86_64/gcc-7.1.0/cdecimal-2.3-72tmlpbrxaawnmjzh2dicis3o6i54jne/include
mkdir -p /blues/gpfs/home/ajstewart/spack/opt/spack/linux-centos7-x86_64/gcc-7.1.0/cdecimal-2.3-72tmlpbrxaawnmjzh2dicis3o6i54jne/lib && cp libmpdec.a libmpdec.so.2.3 /blues/gpfs/home/ajstewart/spack/opt/spack/linux-centos7-x86_64/gcc-7.1.0/cdecimal-2.3-72tmlpbrxaawnmjzh2dicis3o6i54jne/lib
mkdir -p /blues/gpfs/home/ajstewart/spack/opt/spack/linux-centos7-x86_64/gcc-7.1.0/cdecimal-2.3-72tmlpbrxaawnmjzh2dicis3o6i54jne/share/doc/mpdecimal && cp -R doc/* /blues/gpfs/home/ajstewart/spack/opt/spack/linux-centos7-x86_64/gcc-7.1.0/cdecimal-2.3-72tmlpbrxaawnmjzh2dicis3o6i54jne/share/doc/mpdecimal
cp: cannot stat 'doc/*': No such file or directory
make: *** [install] Error 1
```
Do we want a broken package in Spack or should I just drop it?

Reply:
That's me 😄